core/vtk/ttkPointSetToSurface/ttkPointSetToSurface.cpp

```diff
@@ -0,0 +1,171 @@
+#include <ttkPointSetToSurface.h>
+
+#include <vtkCellData.h>
+#include <vtkInformation.h>
+#include <vtkUnstructuredGrid.h>
+
+#include <ttkUtils.h>
+
+#include <array>
+
+vtkStandardNewMacro(ttkPointSetToSurface);
+
+ttkPointSetToSurface::ttkPointSetToSurface() {
+  this->setDebugMsgPrefix("PointSetToSurface");
+
+  this->SetNumberOfInputPorts(1);
+  this->SetNumberOfOutputPorts(1);
+}
+
+int ttkPointSetToSurface::FillInputPortInformation(int port,
+                                                   vtkInformation *info) {
+  if(port == 0) {
+    info->Set(vtkDataObject::DATA_TYPE_NAME(), "vtkPointSet");
+    return 1;
+  }
+  return 0;
+}
+
+int ttkPointSetToSurface::FillOutputPortInformation(int port,
+                                                    vtkInformation *info) {
+  if(port == 0) {
+    info->Set(vtkDataObject::DATA_TYPE_NAME(), "vtkUnstructuredGrid");
```

Comment:
I believe a `vtkPolyData` output would be more appropriate here (better to model 0, 1 and 2D).

Reply:
Done in the next commit!